service/src/main/java/bio/terra/cbas/config/BeanConfig.java

```diff
@@ -32,39 +25,4 @@ public ObjectMapper objectMapper() {
         .setDefaultPropertyInclusion(JsonInclude.Include.NON_ABSENT)
         .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
   }
-
-  /**
-   * Taken from <a
-   * href="https://github.com/DataBiosphere/terra-data-catalog/blob/5cda83aef8548ff98e7cfbe2a6eaaed9ad1bff45/common/src/main/java/bio/terra/catalog/config/BeanConfig.java#L34-L38">Terra
-   * Data Catalog</a> Lasts for the duration of one request, and is injected into dependent beans,
-   * even singletons
-   */
-  @Bean("bearerToken")
-  @RequestScope
-  public BearerToken bearerToken(HttpServletRequest request) {
-    return new BearerTokenFactory().from(request);
-  }
-
-  @Bean("cromwellWriteClient")
-  @RequestScope
```

Comment:
Yes that is correct. These clients were also getting bearer token injected and now instead of that we will be explicitly passing the bearer token to Cromwell or ECM client to set the access token on its API clients.